--- imgSeek-0.8.4/distutils/sysconfig.py.orig	2004-12-27 17:10:57.167834056 +0100
+++ imgSeek-0.8.4/distutils/sysconfig.py	2004-12-27 17:11:26.449382584 +0100
@@ -100,7 +100,7 @@
 
     if os.name == "posix":
         libpython = os.path.join(prefix,
-                                 "lib", "python" + get_python_version())
+                                 "lib64", "python" + get_python_version())
         if standard_lib:
             return libpython
         else:
