--- setup.py~	2003-07-16 21:48:38.000000000 +0200
+++ setup.py	2003-09-23 21:18:55.000000000 +0200
@@ -2,10 +2,10 @@
 
 #############################[ Parameters you should change if install failed ]#########################################
 # qtdir should point to the directory where QT is installed. (Inside this dir you should have a lib and include dir)
-qtdir=["/usr/share/qt/"]      # ["/usr/qt/3/"] if you have a Gentoo system
+qtdir=["/usr/"]      # ["/usr/qt/3/"] if you have a Gentoo system
 
 # python_dir should point to the directory where Python header files may be found.. (Inside this dir you should have a Python.h)
-python_dir="/usr/include/python2.2"
+python_dir="/usr/include/python2.3"
 
 # QT library to use. If qt-mt is not found, you may need to change it to "qt"
 qtlibraries = ["qt-mt"]   # you may want to try "qt" if your QT has no multithreading support
@@ -76,8 +76,8 @@
     if gqtdir not in qtdir: qtdir.append(gqtdir)
     
 for qdir in qtdir:
-    include_dirs.append(qdir+"include")
-    library_dirs.append(qdir+"lib")
+    include_dirs.append(qdir+"include/qt")
+    library_dirs.append(qdir+"lib/qt")
 hasQT=0
 
 for dir in include_dirs:
