--- setup.py.oryg	2004-02-27 21:36:09.000000000 +0000
+++ setup.py	2004-03-23 07:14:43.319579056 +0000
@@ -2,7 +2,7 @@
 
 #############################[ Parameters you should change if install failed ]#########################################
 # qtdir should point to the directory where QT is installed. (Inside this dir you should have a lib and include dir)
-qtdir=["/usr/share/qt/"]      # ["/usr/qt/3/"] if you have a Gentoo system
+qtdir=["/usr/"]      # ["/usr/qt/3/"] if you have a Gentoo system
 
 # python_dir should point to the directory where Python header files may be found.. (Inside this dir you should have a Python.h)
 python_dir="/usr/include/python2.3"
@@ -56,8 +56,8 @@
 """
 else:
     print "Qt is recent enough: "+str(qt.QT_VERSION_STR)
-    include_dirs.append("/usr/include/qt3")
-    library_dirs.append("/usr/lib/qt3")
+    include_dirs.append("/usr/include/qt")
+    library_dirs.append("/usr/lib/qt")
     library_dirs.append("/usr/lib/")
 
 print "#################################### Check QT"
