--- imgSeek-0.8.4/distutils/sysconfig.py.orig	2004-12-27 17:10:57.167834056 +0100
+++ imgSeek-0.8.4/distutils/sysconfig.py	2004-12-27 17:11:26.449382584 +0100
@@ -100,7 +100,7 @@
 
     if os.name == "posix":
         libpython = os.path.join(prefix,
-                                 "lib", "python" + get_python_version())
+                                 "lib64", "python" + get_python_version())
         if standard_lib:
             return libpython
         else:
--- imgSeek-0.8.4/distutils/command/install.py.orig	2004-12-27 17:53:26.848223720 +0100
+++ imgSeek-0.8.4/distutils/command/install.py	2004-12-27 17:53:51.644454120 +0100
@@ -38,15 +38,15 @@
 
 INSTALL_SCHEMES = {
     'unix_prefix': {
-        'purelib': '$base/lib/python$py_version_short/site-packages',
-        'platlib': '$platbase/lib/python$py_version_short/site-packages',
+        'purelib': '$base/lib64/python$py_version_short/site-packages',
+        'platlib': '$platbase/lib64/python$py_version_short/site-packages',
         'headers': '$base/include/python$py_version_short/$dist_name',
         'scripts': '$base/bin',
         'data'   : '$base',
         },
     'unix_home': {
-        'purelib': '$base/lib/python',
-        'platlib': '$base/lib/python',
+        'purelib': '$base/lib64/python',
+        'platlib': '$base/lib64/python',
         'headers': '$base/include/python/$dist_name',
         'scripts': '$base/bin',
         'data'   : '$base',
